engines/script/config.go

```diff
@@ -0,0 +1,60 @@
+package scriptengine
+
+import schematypes "github.com/taskcluster/go-schematypes"
+
+type configType struct {
+	Script     []string `json:"script"`
+	Expiration int      `json:"expiration"`
+	Schema     struct {
+		Type       string                 `json:"type"`
+		Properties map[string]interface{} `json:"properties"`
+		Required   []string               `json:"required"`
+	} `json:"schema"`
+}
+
+var configSchema = schematypes.Object{
+	MetaData: schematypes.MetaData{
+		Title:       "Script Engine Configuration",
+		Description: `Configuration properties for the 'scriptengine'.`,
+	},
+	Properties: schematypes.Properties{
+		"script": schematypes.Array{
+			MetaData: schematypes.MetaData{
+				Title: "Script to Execute",
+				Description: `Script and arguments to execute. This script will be fed
```

Comment:
No, the script is subject to the feeding.. :) hehe

ie. this is `echo "{...}" | script` so the payload is _fed_ to the script.